deltachat-ios/View/ContactCell.swift

```diff
@@ -191,4 +193,43 @@ class ContactCell: UITableViewCell {
     required init?(coder _: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+
+
+    /*
+     this method can be deleted after searchBarContactList-branch was merged into master
```

Comment:
i think, this can just be done by the searchBarContactList-branch, if then still needed.
it's always hard to predict the future :)

Reply:
deleted the comment